package.json

```diff
@@ -11,39 +11,39 @@
   "author": "Adobe Inc.",
   "private": true,
   "dependencies": {
-    "@adobe/aio-lib-ims": "^6.5.0",
-    "@adobe/aio-lib-state": "^2.1.2",
-    "@adobe/aio-sdk": "^4.1.0",
-    "@adobe/exc-app": "^0.2.21",
-    "@adobe/generator-add-action-generic": "^0.2.9",
-    "@adobe/generator-add-web-assets-exc-react": "^0.2.6",
-    "@adobe/generator-app-events-generic": "^1.0.0",
-    "@adobe/react-spectrum": "^3.4.0",
-    "@spectrum-icons/workflow": "^3.2.0",
-    "ajv": "^8.12.0",
+    "@adobe/aio-lib-ims": "^7.0.1",
+    "@adobe/aio-lib-state": "^3.0.2",
+    "@adobe/aio-sdk": "^5.0.1",
+    "@adobe/exc-app": "^1.4.1",
+    "@adobe/generator-add-action-generic": "^1.0.1",
+    "@adobe/generator-add-web-assets-exc-react": "^1.0.2",
+    "@adobe/generator-app-events-generic": "^2.0.1",
+    "@adobe/react-spectrum": "^3.35.1",
+    "@spectrum-icons/workflow": "^4.2.12",
+    "ajv": "^8.16.0",
     "cloudevents": "^8.0.0",
-    "core-js": "^3.6.4",
-    "dotenv": "^16.3.1",
+    "core-js": "^3.37.1",
+    "dotenv": "^16.4.5",
     "got": "^11.8.5",
     "node-fetch": "^2.7.0",
```

Comment:
According to https://developer.adobe.com/runtime/docs/guides/reference/runtimes/#nodejs-v2070, I think we should upgrade this one too

Reply:
By reading the link, I interpret that when using
```
"node-fetch": "3.3.1",
```
you won't need to package it together with the action code. But it is ok to use a different version.

If you don't have big concerns, I'll keep the current version because after upgrading, running 

```bash
npm run test
```

yields the following error 

```
 FAIL  scripts/lib/onboarding.metadata.test.js
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    /Users/rojo/repos/enterprise-integrations/starter-kit-upgrade-to-node20/commerce-integration-starter-kit/node_modules/node-fetch/src/index.js:9
    import http from 'node:http';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      12 |
      13 | jest.mock('node-fetch')
    > 14 | const fetch = require('node-fetch')
         |               ^
      15 | const action = require('../../../scripts/lib/metadata.js')
      16 | const ACCESS_TOKEN = 'token'
      17 | const EMPTY_ENVIRONMENT = {}

      at Runtime.createScriptFromCode (../node_modules/jest-runtime/build/index.js:1505:14)
      at Object.require (scripts/lib/onboarding.metadata.test.js:14:15)
```
I'd like to deal with this issue in a different PR to avoid mixing concerns.

Reply:
sounds good to me!